### DISCUSSION TITLE
staggered face-centered variables

### DISCUSSION THREAD

**Discussion by BenWibking** [Alpine-DAV/ascent#1008] [Q&A] [answered]
Is there support for staggered face-centered variables, such as velocity or magnetic field, or do I need to average these to cell centers before passing them to Ascent?

**Comment by cyrush** [comment, marked as answer]
@BenWibking easiest path is to average in cells. 

There is no face association option for fields.  There are utilities to create derivative typologies to represent faces as elements, but using those will be more complicated than presenting as cell averaged.
